lib/questionnaires/view/src/questionnaire_theme.dart

```diff
@@ -570,9 +570,17 @@ class QuestionnaireThemeData {
     BuildContext context,
     QuestionnaireItemFiller itemFiller,
   ) {
-    return Container(
-      padding: const EdgeInsets.symmetric(horizontal: 16.0),
-      child: itemFiller,
+    return Column(
+      children: [
+        Flexible(
+          child: SingleChildScrollView(
+            child: Container(
+              padding: const EdgeInsets.symmetric(horizontal: 16.0),
+              child: itemFiller,
+            ),
+          ),
+        ),
+      ],
```

Comment:
Is there any reason to have `Column` and `Flexible` here?

We can achieve the same result without having those two widgets.

Reply:
Thanks a lot @hanny-ph, you're right. I've already removed them in the last commit.

I was testing multiple implementations and guess I ended up pushing the first one that actually worked after a few failures.